src/inject.js

```diff
@@ -55,6 +56,49 @@ const stringHandlers = {
             return val;
         }
     },
+
+    // With animationName we look for an object that contains keyframes and
+    // inject them as an `@keyframes` block, returning a uniquely generated
+    // name. The keyframes object should look like
+    //  animationName: {
+    //    from: {
+    //      left: 0,
+    //      top: 0,
+    //    },
+    //    '50%': {
+    //      left: 15,
+    //      top: 5,
+    //    },
+    //    to: {
+    //      left: 20,
+    //      top: 20,
+    //    }
+    //  }
+    // TODO(emily): figure out a way to call this `animation` instead of
+    // `animationName`.
+    animationName: (val) => {
```

Comment:
Well, the CSS key name needs to be `animation-name` in order for it to actually use the animation, and stringHandlers doesn't let us rename what the key is, only change the value. I guess my TODO should be more clear about what doesn't work.